src/components/Provider/ProviderConsumer.tsx

```diff
@@ -16,7 +16,9 @@ export interface ProviderConsumerProps {
 /**
  * The Provider's Consumer is for accessing theme.
  */
-const ProviderConsumer: React.SFC<ProviderConsumerProps> = props => <FelaTheme {...props} />
+const ProviderConsumer: React.SFC<ProviderConsumerProps> = ({ render }) => (
```

Comment:
We had prev here spreading all props, not just the render, not sure if there were other props here, but it may be worth taking a look...